modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/osb-site-initializer-partner-portal-test/src/testFunctional/macros/PRMMDFRequest.macro

```diff
@@ -550,6 +550,28 @@ definition {
 				value1 = "Liferay Brand");
 		}
 
+		if ((${typeOfActivity} == "Content Marketing") && ((${tactic} == "Blog or Article") || (${tactic} == "Business White Paper or eBook") || (${tactic} == "Infographic") || (${tactic} == "Social Content")|| (${tactic} == "Success Story Case Study")|| (${tactic} == "Technical White Paper")|| (${tactic} == "Video")|| (${tactic} == "Web content")|| (${tactic} == "Other"))) {
```

Comment:
Hi @marinaasant0s, as all the fields are the same to all these tactics, you don't need to put all these conditions. Your logic could be just for the typeOfActivity.